pandas/core/arrays/datetimes.py

```diff
@@ -206,45 +206,103 @@ def _simple_new(cls, values, freq=None, tz=None):
         result._tz = timezones.tz_standardize(tz)
         return result
 
-    def __new__(cls, values, freq=None, tz=None, dtype=None):
+    def __new__(cls, values, freq=None, tz=None, dtype=None, copy=False,
+                dayfirst=False, yearfirst=False, ambiguous='raise'):
+        return cls._from_sequence(
+            values, freq=freq, tz=tz, dtype=dtype, copy=copy,
+            dayfirst=dayfirst, yearfirst=yearfirst, ambiguous=ambiguous)
 
-        if freq is None and hasattr(values, "freq"):
-            # i.e. DatetimeArray, DatetimeIndex
-            freq = values.freq
+    @classmethod
```

Comment:
is there a reason you don't want to add ``verify_integrity`` here (as maybe ``_verify_integrity=True``)?

Reply:
We've deprecated the kwarg in the DatetimeIndex constructor to get rid of it.  In cases where verify_integrity is not needed, a different constructor (e.g. simple_new) should be used.